app.py

```diff
@@ -1,12 +1,100 @@
-from flask import Flask, render_template, request, redirect, url_for
-import session_items as session
+from flask import Flask, render_template, request, session, redirect, url_for
 
+_STATUS = ['Not Started', 'In Progress', 'Completed']
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
 
-@app.route('/')
-def index():
-    return 'Hello World!'
+
+@app.route('/', methods=['GET', 'POST'])
+def get_items():
+    items = {}
+    if 'items' in session:
+        items = session['items']
+    return render_template("index.html", data=items)
+
+
+@app.route('/add', methods=['GET', 'POST'])
+def add_item():
+    #statuses = ['Not Started', 'In Progress', 'Completed']
+    items = dict({})
+    if session.get('items'):
+        items = session.get('items')
+
+    if request.method == 'POST':
+        title = request.form['title']
+        description = request.form['description']
+        status = request.form['status']
+        todo_item = crate_item(title, description, status)
+        items[todo_item["id"]] = todo_item
+        session['items'] = items
+        message = " To-DO Item with title '{}' added successfully!".format(title)
+        return render_template("index.html", message=message, data=items)
```

Comment:
This will keep the same URL (which will retrigger this endpoint when the page is refreshed)